static/app/views/profiling/flamegraph.tsx

```diff
@@ -34,76 +20,38 @@ const LoadingGroup: ProfileGroup = {
   profiles: [Profile.Empty()],
 };
 
-interface FlamegraphViewProps {
-  location: Location;
-  params: {
-    eventId?: Trace['id'];
-    projectId?: Project['id'];
-  };
-}
-
-function FlamegraphView(props: FlamegraphViewProps): React.ReactElement {
+function FlamegraphView(): React.ReactElement {
   const organization = useOrganization();
-  const location = useLocation();
-
   const [profileGroup, setProfileGroup] = useProfileGroup();
 
-  const initialFlamegraphPreferencesState = useMemo((): DeepPartial<FlamegraphState> => {
-    return decodeFlamegraphStateFromQueryParams(location.query);
-    // We only want to decode this when our component mounts
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-
   const onImport: ProfileDragDropImportProps['onImport'] = profiles => {
     setProfileGroup({type: 'resolved', data: profiles});
   };
 
   return (
-    <SentryDocumentTitle title={t('Profiling')} orgSlug={organization.slug}>
-      <FlamegraphStateProvider initialState={initialFlamegraphPreferencesState}>
-        <FlamegraphStateQueryParamSync />
-        <Fragment>
-          <Layout.Header>
-            <Layout.HeaderContent>
-              <Breadcrumb
-                location={props.location}
-                organization={organization}
-                trails={[
-                  {type: 'profiling'},
-                  {
-                    type: 'flamegraph',
-                    payload: {
-                      interactionName:
-                        profileGroup.type === 'resolved' ? profileGroup.data.name : '',
-                      profileId: props.params.eventId ?? '',
-                      projectSlug: props.params.projectId ?? '',
-                    },
-                  },
-                ]}
-              />
-            </Layout.HeaderContent>
-          </Layout.Header>
-          <FlamegraphThemeProvider>
-            <FlamegraphContainer>
-              {profileGroup.type === 'errored' ? (
-                <Alert type="error" showIcon>
-                  {profileGroup.error}
-                </Alert>
-              ) : profileGroup.type === 'loading' ? (
-                <Fragment>
-                  <Flamegraph onImport={onImport} profiles={LoadingGroup} />
-                  <LoadingIndicatorContainer>
-                    <LoadingIndicator />
-                  </LoadingIndicatorContainer>
-                </Fragment>
-              ) : profileGroup.type === 'resolved' ? (
-                <Flamegraph onImport={onImport} profiles={profileGroup.data} />
-              ) : null}
-            </FlamegraphContainer>
-          </FlamegraphThemeProvider>
-        </Fragment>
-      </FlamegraphStateProvider>
-    </SentryDocumentTitle>
+    <Fragment>
+      <SentryDocumentTitle
+        title={t('Profiling - Flamegraph')}
```

Comment:
We usually to use `\u2014` instead of `-` because it looks nicer.
```suggestion
        title={t('Profiling \u2014 Flamegraph')}
```